README.md

```diff
@@ -26,22 +26,34 @@ See [example](example) for more details.
 ```javascript
 
 {
-  influxdbUrl: "http://127.0.0.1:8086/INFLUXDB", // INFLUXDB database has to exists
-  measurementName: "MEASUREMENT_NAME",
+  influxdbUrl: 'http://127.0.0.1:8086/INFLUXDB_URL',
```

Comment:
But I just realised that it is not json. Because each key in json has to be in `"` as well as string params.